README.md

```diff
@@ -113,86 +140,99 @@ import ballerina/log;
 import ballerina/mime;
 import ballerina/http;
 
-endpoint http:Listener orderServiceEP {
-    port: 9090
-};
-
-endpoint http:Client circuitBreakerEP {
-
-    // The 'circuitBreaker' term incorporate CB pattern to the client endpoint
-    // Circuit breaker drop remote calls if the endpoint exceeded the failure threshold
-    circuitBreaker: {
-        // Failure calculation window.
-        rollingWindow: {
-            // Time period in milliseconds for which the failure threshold is calculated.
-            timeWindowMillis: 10000,
-            // The granularity at which the time window slides. This is measured in milliseconds.
-            bucketSizeMillis: 2000,
-            // Minimum number of requests in a `RollingWindow` that will trip the circuit.
-            requestVolumeThreshold: 0
-        },
-        // Failure threshold should be in between 0 and 1
-        failureThreshold: 0.2,
-        // Reset timeout for circuit breaker should be in milliseconds
-        resetTimeMillis: 10000,
-        // httpStatusCodes will have array of http error codes tracked by the CB
-        statusCodes: [400, 404, 500]
-    },
-    // HTTP client could be any HTTP endpoint that have risk of failure
-    url: "http://localhost:9092",
-    timeoutMillis: 2000
-};
+listener http:Listener orderServiceListener = new(9090);
+
+http:Client circuitBreakerEP = new("http://localhost:9092", config = {
+        // The 'circuitBreaker' term incorporate circuit breaker pattern to the client endpoint
+        // Circuit breaker will immediately drop remote calls if the endpoint exceeded the failure threshold
+        circuitBreaker: {
+            // Failure calculation window. This is how long Ballerina
```

Comment:
I think this should be fine for the moment. Cause the comments appear in two different places

Reply:
I notice it appears in three places. Eventually it might be a maintenance overhead as well to update comments in multiple places. 

The main concern I have is readability though. I feel a user who reads the first set of comments wouldn't want to read the same thing again. 

Would it make sense to put a note or something explaining that the code in this file is similar to the other file and values, etc. can be set in the same way?

Reply:
I agree this could be a maintenance overhead. However it would make much more sense to have the comments on instead of not putting them at all when illustrating. Also consistency is important. 

If the developers are reading sequentially this would sound like repetition. However if they directly access the section by skipping this would be valuable.

Let's think about this in the next iteration